charts/datadog/templates/_helpers.tpl

```diff
@@ -1045,3 +1045,35 @@ Create RBACs for custom resources
   {{- end -}}
 {{- end -}}
 {{- end -}}
+
+
+{{/*
+  Returns true if Host paths for default OS Release Paths need to be added to the volumes.
+*/}}
+{{- define "should-add-host-path-for-os-release-paths" -}}
+  {{- if ne .Values.targetSystem "linux" -}}
+    false
+  {{- else if .Values.providers.talos.enabled -}}
+    false
```

Comment:
In the PR [#1572](https://github.com/DataDog/helm-charts/pull/1572), you find that in talos the os-release file is present. 
Maybe disabling this mount isn't mandatory when talos?

Reply:
Yes, 
the PR is keeping the host mount for `/etc/os-release` however it removes the mount of other OS specifics release files like `/etc/redhat-release`, `/etc/fedora-release`.